run.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash -ex
```

Comment:
Is this script for local development? The README hasn't been updated so I'm not sure.

It would be great if we could use docker-compose here for dev and test environment - would really cut down on complexity.

We can tackle that in another task though, unless it's related to getting CI pipeline going.